SegFault when running telepresence current-cluster-id

Telepresence will segfault hen running `telepresence current-cluster-id` with telepresence 2.3.6 on MacOS

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xa8 pc=0x1fed182]

goroutine 1 [running]:
k8s.io/cli-runtime/pkg/genericclioptions.(*ConfigFlags).ToRawKubeConfigLoader(0x0, 0x25838c0, 0x0)
	k8s.io/cli-runtime@v0.20.2/pkg/genericclioptions/config_flags.go:122 +0x22
k8s.io/cli-runtime/pkg/genericclioptions.(*ConfigFlags).ToRESTConfig(0x0, 0x0, 0x1, 0x8020103)
	k8s.io/cli-runtime@v0.20.2/pkg/genericclioptions/config_flags.go:115 +0x2b
github.com/datawire/ambassador/pkg/kates.NewClientFromConfigFlags(0x0, 0xc0005ff300, 0xc0005dbcd8, 0xc0005dbd00)
	github.com/datawire/ambassador@v1.13.7-0.20210527054604-663dfb393e59/pkg/kates/client.go:123 +0x32
github.com/telepresenceio/telepresence/v2/pkg/client/cli.ClusterIdCommand.func1(0xc0004eab00, 0x362dbe8, 0x0, 0x0, 0x0, 0x0)
	github.com/telepresenceio/telepresence/v2/pkg/client/cli/cmds_misc.go:28 +0x3f
github.com/spf13/cobra.(*Command).execute(0xc0004eab00, 0x362dbe8, 0x0, 0x0, 0xc0004eab00, 0x362dbe8)
	github.com/spf13/cobra@v1.1.1/command.go:850 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xc0007a0dc0, 0xc00011e000, 0xc0007a0dc0, 0x0)
	github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.1.1/command.go:895
github.com/spf13/cobra.(*Command).ExecuteContext(...)
	github.com/spf13/cobra@v1.1.1/command.go:888
main.main()
	github.com/telepresenceio/telepresence/v2/cmd/telepresence/main.go:31 +0x155
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -17,6 +17,9 @@
   failures causing telepresence to default to the overriding resolver will no
   longer cause general DNS lookup failures.
 
+- Bugfix: Fixed a regression introduced in 2.3.5 that caused `telepresence current-cluster-id`
+  to crash.
+
 ### 2.3.6 (July 20, 2021)
 
 - Bugfix: Fixed a regression introduced in 2.3.5 that caused preview

--- a/pkg/client/cli/cmd.go
+++ b/pkg/client/cli/cmd.go
@@ -136,7 +136,7 @@ func Command(ctx context.Context) *cobra.Command {
 			Name: "Kubernetes flags",
 			Flags: func() *pflag.FlagSet {
 				kubeFlags = pflag.NewFlagSet("", 0)
-				kubeConfig := kates.NewConfigFlags(false)
+				kubeConfig = kates.NewConfigFlags(false)
 				kubeConfig.Namespace = nil // some of the subcommands, like "connect", don't take --namespace
 				kubeConfig.AddFlags(kubeFlags)
 				return kubeFlags

